src/Robo/Commands/Saml/SimpleSamlPhpCommand.php

```diff
@@ -0,0 +1,230 @@
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    return $result;
+  }
+
+  /**
+   * Sets value in project.yml to let targets know simplesamlphp is installed.
+   *
+   * @return \Robo\Result
+   */
+  protected function setSimpleSamlPhpInstalled() {
+    $composerBin = $this->getConfigValue('composer.bin');
+    $projectConfigFile = $this->getConfigValue('blt.config-files.project');
+    $this->say("Updating ${projectConfigFile}...");
+    $result = $this->taskExec("{$composerBin}/yaml-cli update:value")
+      ->arg("${projectConfigFile}")
+      ->arg('simplesamlphp')
+      ->arg('TRUE')
+      ->printOutput(TRUE)
+      ->detectInteractive()
+      ->dir($this->getConfigValue('repo.root'))
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    return $result;
+  }
+
+  /**
+   * Checks if simplesamlphp has already been setup by BLT.
+   *
+   * @return bool
+   */
+  public function isSimpleSamlPhpInstalled() {
+    return $this->getConfig()->has('simplesamlphp') && $this->getConfigValue('simplesamlphp');
+  }
+
+  /**
+   * Creates a symlink from the docroot to the web accessible library dir.
+   *
+   * @return \Robo\Result
+   */
+  protected function symlinkDocrootToLibDir() {
+    $docroot = $this->getConfigValue('docroot');
+
+    $this->say("Creating a symbolic link from ${docroot}/simplesaml to web accessible directory in the simplesamlphp library");
+    $result = $this->taskFileSystemStack()
+      ->symlink("../vendor/simplesamlphp/simplesamlphp/www", "${docroot}/simplesaml")
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    return $result;
+  }
+
+  /**
+   * Copies customized config files into the library on builds.
+   *
+   * @return \Robo\Result
+   *
+   * @command simplesamlphp:build:config
+   */
+  public function simpleSamlPhpBuildConfig() {
+    $this->say('Copying config files to the appropriate place in simplesamlphp library.');
+    $result = $this->taskCopyDir(["{$this->repoRoot}/simplesamlphp" => "{$this->repoRoot}/vendor/simplesamlphp/simplesamlphp"])
+      ->overwrite(TRUE)
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+    $result = $this->taskFileSystemStack()
+      ->copy("{$this->bltRoot}/scripts/simplesamlphp/gitignore.txt", "{$this->repoRoot}/vendor/simplesamlphp/simplesamlphp/.gitignore")
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+
+    return $result;
+  }
+
+  /**
+   * Outputs a message to edit the new config files.
+   */
+  protected function simpleSamlPhpInitComplete() {
+    $docroot = $this->getConfigValue('docroot');
+    $this->say("
+
+
+    ============================================================================
```

Comment:
I'm not crazy about the usage of the `===` as a separator here, since we don't do anything else like this in BLT. I'd prefer to use Symfony Console formatting if necessary. E.g., use `<comment>` or `<info>` wrappers to colorize, and [blocks](http://symfony.com/doc/current/components/console/helpers/formatterhelper.html#print-messages-in-a-block) if necessary.